stdlib/websub/src/main/ballerina/src/websub/subscriber_service_caller.bal

```diff
@@ -24,30 +24,26 @@ public type Caller client object {
 
     # Sends the response to the caller.
     #
-    # + message - The response or any payload of type `string`, `xml`, `json`, `byte[]`,`io:ReadableByteChannel` or `mime:Entity[]`
+    # + message - The response or any payload of type `http:ResponseMessage`
```

Comment:
Shall we just keep the docs as it is? Otherwise user has to look at http:ResponseMessage also to understand the possible types